***Started - reading and parsing pddl file ...***
***Finished reading and parsing pddl file after 0.299 seconds CPU time, 0.289 seconds wall-clock time***

***Started - preprocessing tarski problem ...***
***Finished preprocessing tarski problem after 0.001 seconds CPU time, 0.001 seconds wall-clock time***

***Started - grounding ...***
/tmp/tmp71x57yr2:108:12-13: error: syntax error, unexpected (

/tmp/tmp71x57yr2:109:36-37: error: syntax error, unexpected (

/tmp/tmp71x57yr2:110:44-45: error: syntax error, unexpected (

/tmp/tmp71x57yr2:111:59-60: error: syntax error, unexpected (

/tmp/tmp71x57yr2:112:41-42: error: syntax error, unexpected (

/tmp/tmp71x57yr2:114:26-27: error: syntax error, unexpected (

/tmp/tmp71x57yr2:115:49-50: error: syntax error, unexpected (

/tmp/tmp71x57yr2:116:45-46: error: syntax error, unexpected (

/tmp/tmp71x57yr2:117:34-35: error: syntax error, unexpected (

/tmp/tmp71x57yr2:119:34-35: error: syntax error, unexpected (

/tmp/tmp71x57yr2:120:42-43: error: syntax error, unexpected (

/tmp/tmp71x57yr2:121:57-58: error: syntax error, unexpected (

/tmp/tmp71x57yr2:122:40-41: error: syntax error, unexpected (

/tmp/tmp71x57yr2:124:25-26: error: syntax error, unexpected (

/tmp/tmp71x57yr2:125:50-51: error: syntax error, unexpected (

/tmp/tmp71x57yr2:126:47-48: error: syntax error, unexpected (

/tmp/tmp71x57yr2:127:33-34: error: syntax error, unexpected (

/tmp/tmp71x57yr2:129:50-51: error: syntax error, unexpected (

/tmp/tmp71x57yr2:130:49-50: error: syntax error, unexpected (

/tmp/tmp71x57yr2:131:53-54: error: syntax error, unexpected (

*** ERROR: (gringo): too many messages.
Traceback (most recent call last):
  File "brfs.py", line 60, in <module>
    main( sys.argv[1], sys.argv[2], sys.argv[3] )
  File "brfs.py", line 15, in main
    ground_generate_task( domain_file, problem_file, task )
  File "/home/anubhav-cs/Documents/Research_Project/anubhav-research/LAPKT-public/planners/brfs-tarski/tarski_parser/tarski_util.py", line 185, in ground_generate_task
    reachable_action_params =   copy.deepcopy( grounding.ground_actions())
  File "/media/anubhav-cs/SSD256/Applications/tarski/src/tarski/grounding/lp_grounding.py", line 39, in ground_actions
    model = self._solve_lp()
  File "/media/anubhav-cs/SSD256/Applications/tarski/src/tarski/grounding/lp_grounding.py", line 50, in _solve_lp
    model_filename = run_clingo(lp)
  File "/media/anubhav-cs/SSD256/Applications/tarski/src/tarski/reachability/clingo_wrapper.py", line 26, in run_clingo
    raise ExternalCommandError("Gringo exited with code {}".format(retcode))
tarski.errors.ExternalCommandError: Gringo exited with code 1
